br/pkg/utils/key.go

```diff
@@ -89,3 +95,48 @@ func CompareEndKey(a, b []byte) int {
 
 	return bytes.Compare(a, b)
 }
+
+type KeyRange struct {
+	Start []byte
+	End   []byte
+}
+
+// encode
+func formatAPIV2Key(key []byte, isEnd bool) []byte {
+	if isEnd && len(key) == 0 {
+		return []byte{APIV2KeyPrefixEnd}
+	}
+	apiv2Key := []byte{APIV2KeyPrefix}
+	return append(apiv2Key, key...)
+}
+
+// FormatAPIV2KeyRange convert user key to APIV2 format.
```

Comment:
Suggest to add unit tests for the following two functions.

Reply:
test cased are added.